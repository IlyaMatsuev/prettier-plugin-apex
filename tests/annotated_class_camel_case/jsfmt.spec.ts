runSpec(__dirname, ["apex"], { apexAnnotationsCamelCase: true });
