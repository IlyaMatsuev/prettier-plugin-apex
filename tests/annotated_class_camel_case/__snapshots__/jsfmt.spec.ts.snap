// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Format apex: AnnotatedClassCamelCase.cls: AnnotatedClassCamelCase.cls 1`] = `
@isTest class AnnotatedClassCamelCase
{}

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@IsTest
class AnnotatedClassCamelCase {
}

`;
